server/public/shared/mlog/levels.go

```diff
@@ -48,4 +55,9 @@ var (
 // Combinations for LogM (log multi).
 var (
 	MLvlAuditAll = []Level{LvlAuditAPI, LvlAuditContent, LvlAuditPerms, LvlAuditCLI}
+
+	MlvlLDAPError = []Level{LvlError, LvlLDAPError}
+	MlvlLDAPWarn  = []Level{LvlWarn, LvlLDAPWarn}
+	MlvlLDAPInfo  = []Level{LvlInfo, LvlLDAPInfo}
+	MlvlLDAPDebug = []Level{LvlDebug, LvlLDAPDebug}
```

Comment:
Error, warn, info and debug messages still get sent to the main log file. Otherwise, this PR would be a breaking change. We might want to revisit that decision in a major release and only send errors and warnings to the main log file.